netzob/src/netzob/Import/PCAPImporter/all.py

```diff
@@ -35,7 +35,7 @@
 try:
     import pcapy
     pcapy_available = True
-except ImportError:
+except ImportError:    
```

Comment:
There some areas where unecessary spaces are added at the end of the line in this PR (maybe because of the underlying editor ?). I would suggest to use a tool (such as autopep8, yapf, or pep8) to remove uncessary spaces.